lib/services/local/resource_test.go

```diff
@@ -230,3 +230,22 @@ func newUserTestCase(t *testing.T, name string, roles []string, withSecrets bool
 	}
 	return &user
 }
+
+func TestBootstrapLock(t *testing.T) {
+	t.Parallel()
+	ctx := context.Background()
+	tt := setupServicesContext(ctx, t)
+
+	nl, err := types.NewLock("test", types.LockSpecV2{
+		Target: types.LockTarget{
+			User: "user",
+		},
+		Message: "lock test",
+	})
+	require.NoError(t, err)
+	require.NoError(t, CreateResources(ctx, tt.bk, nl))
+
+	l, err := tt.suite.Access.GetLock(ctx, "test")
+	require.NoError(t, err)
+	require.Empty(t, cmp.Diff(nl, l))
```

Comment:
Maybe:

```suggestion
	require.Empty(t, cmp.Diff(nl, l, protocmp.Transform()))
```

(Transform is https://pkg.go.dev/google.golang.org/protobuf/testing/protocmp#Transform)

This avoids occasional failures due to differences in internal (XXX_) fields. protocmp is not meant for Gogo, but still I think it works. Give it a go.

Reply:
Thanks. added in https://github.com/gravitational/teleport/pull/27168/commits/20e273cc9af83ff98d52ee56381e21389e4b8a1f